src/test/java/space/obminyashka/items_exchange/end2end/LocationFlowTest.java

```diff
@@ -99,7 +99,51 @@ private void sendAndCompareLocationResponse(MockHttpServletRequestBuilder reques
                 .andExpect(jsonPath("$.areaUA").value(AREA_UA))
                 .andReturn();
     }
-
+    @WithMockUser(username = "admin", roles = { "ADMIN" })
+    @Test
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "location/create.yml", orderBy = "city_en", ignoreCols = "id")
+    void createTwoSameLocation_shouldGetMessageBadRequest() throws Exception
```

Comment:
Valid test class name syntax in our case "testMethod_should..._when". I'd be very grateful if you rename all your created test methods according to it.